### PR TITLE
Updated Utils

### PR DESCRIPTION
**Pull request by Sahiltheram**

### CODE REVIEW

File: src/main/java/frc/robot/utils/diag/DiagPot.java [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (armadilloz) [inline src/main/java/frc/robot/utils/diag/DiagPot.java:2]: Content missing?

File: utils/ColorSensor.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
+/*----------------------------------------------------------------------------*/
```
Comment (armadilloz) [inline utils/ColorSensor.java]: Duplicate file

File: utils/MotorUtils.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
+/*----------------------------------------------------------------------------*/
```
Comment (armadilloz) [inline utils/MotorUtils.java]: Duplicate file. This is the same for all the files under utils

**Review by armadilloz — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/utils/diag/DiagPot.java (hunk 1 above)
Comment (batchen1) [reply src/main/java/frc/robot/utils/diag/DiagPot.java:2]: added missing content

File: utils/ColorSensor.java (hunk 2 above)
Comment (batchen1) [reply utils/ColorSensor.java]: removed

File: utils/MotorUtils.java (hunk 3 above)
Comment (batchen1) [reply utils/MotorUtils.java]: removed

**Review by armadilloz — APPROVED** [review]

**Pull request merged**